pennylane/ops/op_math/controlled_decompositions.py

```diff
@@ -158,6 +158,8 @@ def ctrl_decomp_zyz(target_operation: Operator, control_wires: Wires):
 
     .. code-block:: python
 
+        import pennylane as qml
```

Comment:
pennylane and numpy imports are implicit as discussed yesterday, so this isn't necessary, but it's harmless.